opendbc/safety/tests/test_byd.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+import unittest
+import opendbc.safety.tests.common as common
+from opendbc.car.structs import CarParams
+from opendbc.safety.tests.libsafety import libsafety_py
+from opendbc.safety.tests.common import CANPackerPanda
+
+
+class TestBydSafety(common.PandaCarSafetyTest, common.AngleSteeringSafetyTest):
+
+  TX_MSGS = [[0x1E2, 0], [0x316, 0]]
+  STANDSTILL_THRESHOLD = 0
+  GAS_PRESSED_THRESHOLD = 3
+  RELAY_MALFUNCTION_ADDRS = {0: (0x1E2, 0x316)}
+  FWD_BLACKLISTED_ADDRS = {0: [], 2: [0x1E2, 0x316]}
+  FWD_BUS_LOOKUP = {0: 2, 2: 0}
+
+  MAIN_BUS = 0
+  CAM_BUS = 2
```

Comment:
We don't usually split this out into a constant since 0/2 is very often the main and cam bus